ex3/index.js

```diff
@@ -0,0 +1,18 @@
+const readline = require('readline');
+const Ireadline = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+const strangePerson = 'Estranho'
+
+Ireadline.question('Qual o seu nome? ', function(name){
+  
+  if(name == 'Marcus'|| name == 'Victor'){
+    console.log(`Olá, ${name}!`);
+  }
+  else{
+    console.log(`Olá, ${strangePerson}!`)
+  }
+    
+  Ireadline.close();
+});
```

Comment:
Like this:

```suggestion
Ireadline.question('Qual o seu nome? ', (name) => {
  let personsName = "Estranho";
  
  if(name == 'Marcus' || name == 'Victor') {
    personsName = name;
  }
  
  console.log(`Olá, ${name}!`);
  Ireadline.close();
});
```

This way we can get rid of the "else" clause and we consistently print out to the console the same variable.